lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -87,6 +88,12 @@
     state: import
     target: /tmp/dump.sql.bz2
 
+- name: Dump multiple databases(not all) to hostname.sql
+  mysql_db:
+    state: dump
+    name: db_1, db_2
+    target: /tmp/{{ inventory_hostname }}.sql
```

Comment:
```suggestion
    target: /tmp/dump.sql
```
Adding an extra variable in the example may mislead the reader on how to use the module

Reply:
This variable has been used in other examples as well.
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/database/mysql/mysql_db.py#L73

To keep consistency, I thought of using the same here, however, removing it now